src/Controller/QueueTrigger.php

```diff
@@ -24,14 +25,17 @@ public function processToEndTime() {
       throw new NotFoundHttpException();
     }
     else {
-      // The endpoint has been called successfully.
-      // @Todo Call the queue runner service.
-      // Log it.
-      $vars = ['@time' =>  date('m-d-Y H:i:s', $time)];
+      // Process our bulletins.
+      $trigger = new BulletinBase();
+      $status = $trigger->processQueue($time);
+
+      // The endpoint has been called.
+      // Log status.
+      $vars = ['@time' => date('m-d-Y H:i:s', $time)];
```

Comment:
```suggestion
      $vars = [
        '@time' => date('m-d-Y H:i:s', $time),
        '@status' => $status,
        ];
```

See changes farther down.

Reply:
Sorry about that - wasn't paying attention. :+1: